quickstarts/Embeddings.ipynb

```diff
@@ -35,18 +35,34 @@
     },
```

Comment:
<p>Use <code>output_dimensionality=4</code> here</p><p> </p><p>Include a comment explaining it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>(That way you wouldn't need the '... trimmed')</p>